### ISSUE TITLE
Removed by mod tag

### ISSUE THREAD

**Issue by slrgt** [aeharding/voyager#1213]
**Is your feature request related to a problem? Please describe.**
When a post is removed by a mod or deleted voyager doesn't list that fact

**Describe the solution you'd like**
It would be convenient if Removed By Mod appeared on the thread in voyager similar to how it does on the web ui of lemmy

**Comment by aeharding** [comment]
Are you looking at lemmy 0.19 instance?

Also a reproduction case is needed. Thanks

**Comment by slrgt** [comment]
Yes and what do you mean by reproduction case? Like how to see this in action?

Create a thread, have it removed by a mod, go to the thread on lemmy web ui, it’ll say Removed by Mod, go to the thread on Voyager, it wont mention its been Removed by Mod

**Comment by aeharding** [comment]
Yes exactly, just a link where I can see it not working.

Voyager should show as removed by mod, so I've tagged this as a bug.

**Comment by slrgt** [comment]
Heres an example https://lemmy.ml/post/11437227

**Comment by slrgt** [comment]
Oh the link doesnt work because it has to be your thread that is removed. Other people

**Comment by aeharding** [comment]
Ahh, I see. That definitely sounds like a bug then, thanks for reporting!
